search.py

```diff
@@ -27,5 +27,6 @@ def find_match(city: Model):
 
 
 if __name__ == "__main__":
-    query = sys.argv[1]
-    print(search(query))
+    src, query, populate = sys.argv[1:]
+    populate = bool(populate)
+    print(search(src, query, populate))
```

Comment:
```suggestion
    print(json.dumps(search(src, query, populate), ensure_ascii=False, indent=4))
```

Reply:
Thank you very much for your fix. I've made some changes and forget to update this file. I suggest adding json.dumps also in order to preserve indent.